pints/_mcmc/_slice_covariance_matching.py

```diff
@@ -0,0 +1,354 @@
+# -*- coding: utf-8 -*-
+#
+# Slice Sampling - Covariance Adaptive: Covariance Matching MCMC Method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+
+
+class SliceCovarianceMatchingMCMC(pints.SingleChainMCMC):
+    """
```

Comment:
Please use ``` r""" ``` to create a "raw string". See here https://github.com/pints-team/pints/issues/735